ivy/functional/frontends/numpy/manipulation_routines/changing_kind_of_array.py

```diff
@@ -1,6 +1,6 @@
 # local
-import ivy
+from ivy.functional.frontends.numpy.creation_routines import asarray as as_array
 
 
 def asarray(a, dtype=None, order=None, *, like=None):
-    return ivy.asarray(a, dtype=dtype)
```

Comment:
we can delete this function definition entirely, importing like so at the top of the file is enough to ensure that the function is included in this namespace:

```
from ivy.functional.frontends.numpy.creation_routines import asarray
```